department-of-veterans-affairs/abd-vro
# Add Expanded lookup table endpoint

## What was the problem?
This implements a new classification lookup table to classify Veteran contentions.  The research for this new method is in [Free Text Classification Research](https://docs.google.com/document/d/19SAPn-fHwXT8OQqBu-UYoW0YQl9W7rwrE7bVNv4hgJA/edit?tab=t.0#heading=h.dne1lrf9ar3e).

This establishes the new method and adds the endpoint. This will not be implemented in vets-api until we are able to conduct testing on the processing time and load testing on the API to ensure that it does not slow down the full submit process for the 526EZ.  

Associated tickets or Slack threads:
- [#626](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/626)

## How does this fix it?[^1]


## How to test this PR
- From cc-app directory run `pytest`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
